src/System.Net.Security/tests/UnitTests/System/Security/Authentication/ExtendedProtection/ExtendedProtectionPolicyTest.cs

```diff
@@ -14,36 +14,49 @@ public class ExtendedProtectionPolicyTest
         [Fact]
         public void Constructor_PolicyEnforcement_NeverParam()
         {
-            var ex = AssertExtensions.Throws<ArgumentException>("policyEnforcement", () => new ExtendedProtectionPolicy(PolicyEnforcement.Never, ProtectionScenario.TransportSelected, null));
-            Assert.Equal(typeof(ArgumentException), ex.GetType());
+            AssertExtensions.Throws<ArgumentException>("policyEnforcement", () => new ExtendedProtectionPolicy(PolicyEnforcement.Never, ProtectionScenario.TransportSelected, null));
         }
 
         [Fact]
         public void Constructor_ServiceNameCollection_ZeroElementsParam()
         {
-            var paramValue = new ServiceNameCollection(new List<string>());
-            var ex = AssertExtensions.Throws<ArgumentException>("customServiceNames", () => new ExtendedProtectionPolicy(PolicyEnforcement.Always, ProtectionScenario.TransportSelected, paramValue));
-            Assert.Equal(typeof(ArgumentException), ex.GetType());
+            ServiceNameCollection paramValue = new ServiceNameCollection(new List<string>());
+            AssertExtensions.Throws<ArgumentException>("customServiceNames", () => new ExtendedProtectionPolicy(PolicyEnforcement.Always, ProtectionScenario.TransportSelected, paramValue));
+        }
+
+        [Fact]
+        public void Constructor_PolicyEnforcementChannelBinding_NeverParam()
+        {
+            MockCustomChannelBinding customChannelBinding = new MockCustomChannelBinding();
+            AssertExtensions.Throws<ArgumentException>("policyEnforcement", () => new ExtendedProtectionPolicy(PolicyEnforcement.Never, customChannelBinding));
         }
 
         [Fact]
         public void Constructor_ChannelBinding_NullParam()
         {
-            var ex = AssertExtensions.Throws<ArgumentNullException>("customChannelBinding", () => new ExtendedProtectionPolicy(PolicyEnforcement.Always, null));
-            Assert.Equal(typeof(ArgumentNullException), ex.GetType());
+            AssertExtensions.Throws<ArgumentNullException>("customChannelBinding", () => new ExtendedProtectionPolicy(PolicyEnforcement.Always, null));
         }
 
         [Fact]
         public void Constructor_CollectionParam()
         {
-            var paramValue = new List<string> { "Test1", "Test2" };
-            var expectedServiceNameCollection = new ServiceNameCollection(paramValue);
-            var extendedProtectionPolicy = new ExtendedProtectionPolicy(PolicyEnforcement.Always, ProtectionScenario.TransportSelected, paramValue);
+            List<string> paramValue = new List<string> { "Test1", "Test2" };
+            ServiceNameCollection expectedServiceNameCollection = new ServiceNameCollection(paramValue);
+            ExtendedProtectionPolicy extendedProtectionPolicy = new ExtendedProtectionPolicy(PolicyEnforcement.Always, ProtectionScenario.TransportSelected, paramValue);
```

Comment:
all 3 of these can be left as 'var'.

'var is used when the type is obvious such as from a 'new' construct.